src/getHumanAge.test.js

```diff
@@ -1,10 +1,60 @@
 'use strict';
 
-describe('getHumanAge', () => {
+describe(`getHumanAge`, () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
+  test(`should be declared`, () => {
```

Comment:
Use single quotes instead of backticks for strings without template literals or variables. Replace `should be declared` with 'should be declared'. 📝